jobs/release.Jenkinsfile

```diff
@@ -223,7 +223,10 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
                                           credentialsId: 'oscontainer-push-registry-secret')]) {
                         def repo = registry_repos[configname]
                         def (artifact, metajsonname, tag_suffix) = val
-                        def arch_args = basearches.collect{"--arch ${it}"}.join(" ")
+                        def extra_args = basearches.collect{"--arch ${it}"}.join(" ")
+                        if (registry_repos.v2s2) {
+                            extra_args += " --v2s2"
+                        }
```

Comment:
I wonder if it would be worth changing this code to use a extra_args list and then do the `.join(' ')` in the shwrap call below. like we do with `fetch_args` in the `Fetch Artifacts` stage above. Prevents us having to handle whitespace when we define the args.

Reply:
Added a commit for that!